src/stack/index.ts

```diff
@@ -0,0 +1 @@
+export * from './stack';
```

Comment:
It's a matter of convenience. Its purpose is to simplify work while dealing with import/export. Using that I only need to provide the folder path, not a separate file:
```javascript
import { Stack } from './stack'
```
So that if I'm going to rename stack.ts, split it into several files, or add some new files I'll only edit the index.ts file and there is no need to update my imports where ever they are.